includes/html/table/arp-search.inc.php

```diff
@@ -1,121 +1,104 @@
 <?php
```

Comment:
If you are going to substantially re-write this it would be better to put it under a Laravel route (instead of a legacy one) Like App\Http\Controllers\Table